NEWS.md

```diff
@@ -91,6 +91,8 @@ Library improvements
     * Rank one update and downdate functions, `lowrankupdate`, `lowrankupdate!`, `lowrankdowndate`,
     and `lowrankdowndate!`, for dense Cholesky factorizations ([#14243],[#14424])
 
+    * All `sparse` methods now retain provided numerical zeros as structural nonzeros.
```

Comment:
Sounds wise. In fact, given `dropzeros!` should become increasingly important / regularly used with the shift to retaining numerical zeros by default, perhaps `dropzeros!` should now be exported?